ConsistencyManager/Protocols/ConsistencyManagerModel.swift

```diff
@@ -188,3 +205,15 @@ public extension ConsistencyManagerModel where Self: Equatable {
         }
     }
 }
+
+public extension ConsistencyManagerModel {
+    func mergeModel(model: ConsistencyManagerModel) -> ConsistencyManagerModel? {
+        // Usually, we don't need to merge and instead just return the other model
```

Comment:
No, this is it. Most people dont' use projections and this is what they want. If you use projections, you'll need to implement this yourself.